tendme.lic

```diff
@@ -26,7 +26,7 @@ class TendMe
       health_data['parasites'].values.flatten +
       health_data['bleeders'].values.flatten
     )
-    .reject { |wound| wound.internal? || wound.bleeding_rate =~ /tended|clotted/ } # untendable
+    .reject { |wound| wound.internal? || wound.bleeding_rate =~ /tended|clotted/ || wound.body_part == 'skin' } # untendable
```

Comment:
Should 'tendable?' be a wound class property so this doesn't have to go everywhere?

Reply:
Yeah, I debated that too. I was just being lazy. I'll add a tendable property on the wound class.